client/test/e2e/playwright/auth.setup.ts

```diff
@@ -1,21 +1,65 @@
+/* eslint-disable no-param-reassign */
 import { test as setup, expect } from '@playwright/test';
+import * as dotenv from 'dotenv';
+
+dotenv.config({ path: './test/.env' });
 
 const authFile = 'playwright/.auth/user.json';
+const testUsername = process.env.REACT_APP_TEST_USERNAME ?? '';
+const testPassword = process.env.REACT_APP_TEST_PASSWORD ?? '';
 
 setup('authenticate', async ({ page }) => {
   // Perform authentication steps for authentication process.
   await page.goto('./');
   await page
     .getByRole('button', { name: 'GitLab logo Sign In with GitLab' })
     .click();
-  await page.getByRole('button', { name: 'Accept All Cookies' }).click();
-  await page.fill('#user_login', 'TestUserDTaaS'); // Insert valid GitLab testing username.
-  await page.fill('#user_password', ''); // Insert valid GitLab testing password.
+  try {
+    await page.waitForSelector('label[for="user_login"]', {timeout: 30000}); // wait up to 4 seconds
```

Comment:
should it be 30 seconds in the comment?

Reply:
Yes sorry this was because I was finish up the code on my laptop which is super slow, I've changed it now to 10 secs which seems more reasonable.